### PR TITLE
Wine reviews

### PR DESCRIPTION
**Pull request by TomSoucie**
#### What does this PR do?
This PR completes functionality for a user can review a wine and venue, as a polymorphic relationship.
#### Where should the reviewer start?
Reviewer should start in the "user_wants_to_review_something_spec" to identify app functionality requirements, and then check the reviews_controller, venue views, wine views and the relationships established in the User, Wine and Venue models and the migration/schema
#### How should this be manually tested?
-Start rails server
-Login as a user
-Navigate to a wine and venue show page
-Leave a review!
#### Any background context you want to provide?
Polymorphic relationship allows the use of a Reviews table for wines and venues

#### What are the relevant tickets?
[Story #148705185](https://www.pivotaltracker.com/story/show/148705185)
[Story #148705179](https://www.pivotaltracker.com/story/show/148705179)
#### Screenshots (if appropriate)
N/A
#### Questions:
  - Do Migrations Need to be ran?
Yes - rake db:drop and rake db:{create,migrate,seed}
  - Do Environment Variables need to be set?
No
  - Any other deploy steps?
Push to Heroku with 'push Heroku master' after verifying all tests pass
[Finishes #148705185][Finishes #148705179]

### CODE REVIEW

File: app/controllers/user_follows_controller.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class UserFollowsController < ApplicationController
+  def new
+    @venue = Venue.new
+  end
+
+  def create
+    binding.pry
```
Comment (alex-w-k) [inline app/controllers/user_follows_controller.rb]: Is this pry meant to be there?

**Review by alex-w-k — CHANGES_REQUESTED** [review]
Looks good other than the pry

File: app/models/user.rb [hunk 2]
```diff
@@ -9,7 +9,10 @@ class User < ApplicationRecord
   has_many :user_venues, as: :manager
   has_many :venues, through: :user_venues
 
+  has_many :reviews, as: :reviewable
+
   def manager_has_venues_with_wine?(wine)
     !(venues & wine.venues).empty?
```
Comment (iamchrissmith) [inline app/models/user.rb]: Do you know why this is failing the Travis build?

**Review by iamchrissmith — CHANGES_REQUESTED** [review]
Take a look at Travis and see why the Manager has venue with wine method is failing

File: app/controllers/user_follows_controller.rb (hunk 1 above)
Comment (TomSoucie) [reply app/controllers/user_follows_controller.rb]: Nope - I'll take a look, thanks

File: app/models/user.rb (hunk 2 above)
Comment (TomSoucie) [reply app/models/user.rb]: No, I'll check into this

File: app/controllers/user_follows_controller.rb (hunk 1 above)
Comment (TomSoucie) [reply app/controllers/user_follows_controller.rb]: OK this is removed but still failing Travis build, I'll keep looking into it

File: app/models/user.rb (hunk 2 above)
Comment (TomSoucie) [reply app/models/user.rb]: Took out "as reviewable" to see if there was some interaction there, still failing. I'll keep trying to debug

**Review by MasonHolland — APPROVED** [review]

**Pull request merged**